wagtailannotatedimage/static/annotated-image/annotated-image.css

```diff
@@ -19,9 +19,12 @@ span.note {
 
 .annotated-image--container {
 	position: relative;
-	height: 100%;
-	width: 100%;
+    display: inline-block;
```

Comment:
Pedantry: mixed tabs and spaces here. Perhaps add an `.editorconfig` to enforce a consistent style?